requirements/conda.txt

```diff
@@ -19,4 +19,4 @@ qdarkstyle >=2.6.4
 atomicwrites
 diff-match-patch
 pexpect
-
+watchdog
```

Comment:
Please also add `watchdog` to our `setup.py` and our Readme.

Reply:
Should we add this also to the `requirements.txt?`